rmldatatfbbox/sample_configs/bbox_all_fields.yaml

```diff
@@ -0,0 +1,19 @@
+# This sample config contains all fields supported by ravenML core and the bbox plugin.
+# Plugin specific configuration is located in the plugin field.
+
+imageset: # Can provide a list of imagesets if needed
+  - jigsaw_tester
+overwrite_local: True
+dataset_path: '~/Desktop/test'
+# filter: True
+upload: False
+delete_local: True
+metadata:
+    created_by: Abhi Dhir
+    comments: test
+plugin:
+    verbose: True
+    kfolds: 5
+    dataset_name: bbox_test_dataset_abhi
+    test_percent: .2 # optional
```

Comment:
test_percent is something I added since it might be useful to modify from the default (.2) but is also optional (won't prompt if not provided)